package.json

```diff
@@ -27,7 +27,8 @@
     "@chakra-ui/react": "^2.4.9",
     "@emotion/react": "^11.8.1",
     "@emotion/styled": "^11.8.1",
-    "framer-motion": "^8.5.2",
+    "framer-motion": "^10.13.0",
+    "jest-environment-jsdom": "^29.6.1",
```

Comment:
jest-environment-jsdom needs to be under `devDependency` and not under `dependency` filed

Reply:
Addressed the comment. Moved "jest-environment-jsdom" from `dependency` to `devDependency`.